test/acceptance/e2e/cluster-aggr/scenario.js

```diff
@@ -1,8 +1,6 @@
 const map = new carto.Map({
     container: 'map',
-    background: 'black',
-    center: [-75, 40],
```

Comment:
this was set to avoid draw ordering issues

Reply:
But this test uses `carto.Map` that has no `center`, `zoom` attributes...

Reply:
ah, true, yes, I probably copy pasted them from the example I was thinking about